image-meta-tag/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+    name: image-meta-tag
+    version: "0.4"
+
+source:
+    git_url: https://github.com/SciTools-incubator/image-meta-tag
+    git_tag: 0.4.1
+
+build:
+    number: 0
```

Comment:
But in any case, all builds failed, and it appears to be because Continuum was not serving up Miniconda correctly.